rustfmt.toml

```diff
@@ -1 +1,3 @@
 reorder_imports = true
+reorder_import_names = true
```

Comment:
This was already done by a previous PR, can you remove this commit from the branch?